srcpkgs/corosync/template

```diff
@@ -0,0 +1,64 @@
+# Template file for 'corosync'
+pkgname=corosync
+version=3.0.2
+revision=1
+build_style=gnu-configure
+configure_args="--disable-static --enable-dbus --enable-monitoring
+ --enable-nozzle --enable-snmp --enable-xmlconf --enable-vqsim
+ --with-logdir=/var/log/corosync"
+make_dirs="/etc/corosync/uidgid.d 0755 root root
+ /etc/corosync/service.d 0755 root root
+ /var/log/corosync 0755 root root
+ /var/lib/corosync 0755 root root"
+hostmakedepends="autoconf automake libtool pkg-config"
+makedepends="dbus-devel libknet1-devel libnozzle1-devel libqb-devel
+ libstatgrab-devel libtool net-snmp-devel nss-devel"
+short_desc="Cluster engine daemon and utilities"
+maintainer="Jiří Bělka <jirib79@gmail.com>"
+license="BSD-2-Clause"
+homepage="http://corosync.github.io/corosync"
+distfiles="https://github.com/corosync/corosync/releases/download/v${version}/${pkgname}-${version}.tar.gz"
```

Comment:
This is the wrong format for this link.  Please review other packages and see how the github links should look, which will subsequently allow you to drop the update file below.